example/aggregate/postcard/postcard.go

```diff
@@ -20,9 +21,8 @@ type Postcard struct {
 
 func NewPostcard(id string) (*Postcard, error) {
 	p := &Postcard{}
-	p.eq = aggregate.NewEventsQueue(p)
 
-	err := p.eq.PushAndApply(&Created{
+	err := aggregate.Record[*Postcard](p, &p.eq, &Created{
```

Comment:
It caught my attention... What do you think about the name of the package `aggregate`? Given that this is the central lib package, maybe we could have here just `esja`, `es`, or `eventsourcing`. The specific packages like `sql` and `pii` are fine in my opinion.

Reply:
@roblaszczak @maclav3 thoughts?